src/main/java/com/internet/shop/model/Order.java

```diff
@@ -0,0 +1,39 @@
+package com.internet.shop.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Order {
+    private Long id;
+    private List<Product> products = new ArrayList<>();
+    private Long userId;
+
+    public Order(List<Product> products, Long userId) {
+        this.products = products;
+        this.userId = userId;
+    }
```

Comment:
```suggestion
public class Order {
    private Long id;
    private List<Product> products;
    private Long userId;

    public Order(Long userId) {
        products = new ArrayList<>();
        this.userId = userId;
    }
```
same with shopping cart